cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,112 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
+
+  it('Should allow to add a new worker', () => {
+    cy.visit('');
+    cy.addUser(user, 1);
+    cy.assertUser(user);
+  });
+
+  it('Should allow to delete a worker', () => {
+    cy.visit('');
+    cy.addUser(user, 1);
+    cy.get('.rt-tbody')
+      .should('contain', user.firstName);
+    cy.get('#delete-record-4').click();
+    cy.get('.rt-tbody')
+      .should('not.contain', user.firstName);
+  });
+
+  it('Should allow to delete all workers', () => {
+    cy.visit('');
+    cy.get('h1').should('contain', 'Web Tables');
+    cy.get('#delete-record-1').click();
+    cy.get('.rt-tbody')
+      .should('not.contain', 'Cierra');
+    cy.get('#delete-record-2').click();
+    cy.get('.rt-tbody')
+      .should('not.contain', 'Alden');
+    cy.get('#delete-record-3').click();
+    cy.get('.rt-tbody')
+      .should('not.contain', 'Kierra');
+    cy.contains('No rows found').should('exist');
+  });
+
+  it('Should allow to find a worker in the search field and edit it', () => {
```

Comment:
you create the user here and edit it with the same values. For example you edit first name "Yuriy" with first name "Yuriy". Change the values for editing